segments/gcalcli.sh

```diff
@@ -0,0 +1,32 @@
+# Print next Google Calendar Event
+
+TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME_DEFAULT=1
+
+generate_segmentrc() {
+	read -d '' rccontents  << EORC
+gcalcli uses military time by default - if you want to see 12 hour time format, set TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME_DEFAULT to 0
+export TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME="${TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME_DEFAULT}"
+EORC
+	echo "$rccontents"
+}
+
+__process_settings() {
+	if [ -z "$TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME" ]; then
```

Comment:
I would change military time to “24hr” or similar, as I’m quite sure this is more common term to use (in the software business at least).

Reply:
That's a good point, I've updated this 👍🏻